bundles/org.openhab.core/src/main/java/org/openhab/core/types/util/UnitUtils.java

```diff
@@ -112,6 +113,10 @@ public class UnitUtils {
      * @return The Dimension string or null if the unit can not be found in any of the SystemOfUnits.
      */
     public static @Nullable String getDimensionName(Unit<?> unit) {
+        // Special cased, because the actual dimension is 1/Temperature
```

Comment:
Hmm. Rather than making a special case here, I wonder if we should not try to make a 'proper' fix to `systemUnit.isCompatible(unit)` on line 137 below. ??

Reply:
You're absolutely right -  it should check if the inverse unit is compatible, just like QuantityType.toInvertibleUnit

Reply:
```
} else if (compatibleDimension == null && (systemUnit.isCompatible(unit) || systemUnit.isCompatible(unit.inverse()))) {
                                compatibleDimension = dimension;
}
```

Reply:
and Junit test..

```
    @Test
    void testGetDimensionNameForInvertibleUnit() {
        assertEquals("Temperature", UnitUtils.getDimensionName(Units.MIRED));
        assertEquals("Temperature", UnitUtils.getDimensionName(Units.KELVIN));
    }
```